modular_skyrat/modules/goofsec/code/clothing/security_head.dm

```diff
@@ -0,0 +1,67 @@
+/**
+ * Standard security helmet
+*/
+/obj/item/clothing/head/helmet/sec
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "security_helmet"
+	toggle_message = "You pull the visor down on"
+	alt_toggle_message = "You push the visor up on"
+	actions_types = list(/datum/action/item_action/toggle)
+	can_toggle = TRUE
+	toggle_cooldown = 0
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH | PEPPERPROOF
+	visor_flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH | PEPPERPROOF
+
+/**
+ * Bullet proof helmet
+ */
+/obj/item/clothing/head/helmet/alt
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "bulletproof_helmet"
+
+/**
+ * Security beret - now garrison
+*/
+/obj/item/clothing/head/security_garrison
+	name = "security garrison cap"
+	desc = "A robust garrison cap with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "garrison_black"
+	uses_advanced_reskins = TRUE
+	armor = list(MELEE = 30, BULLET = 25, LASER = 25, ENERGY = 35, BOMB = 25, BIO = 0, FIRE = 20, ACID = 50)
+	strip_delay = 60
+	mutant_variants = NONE
+	unique_reskin = list(
+		"Blue Variant" = list(
+			RESKIN_ICON_STATE = "garrison_blue",
+			RESKIN_WORN_ICON_STATE = "garrison_blue"
+		),
+	)
+
+/**
+ * Security cap
+*/
+/obj/item/clothing/head/security_cap
```

Comment:
out of curiosity, why isnt this a subtype of /obj/item/clothing/head/soft? it'd let it be worn backwards if so, and fundamentally nothing else would change

Reply:
Too unprofessional tbh, and I don't want to reverse the sprites etc.